labs/lab1/chapter1.md

```diff
@@ -1,10 +1,10 @@
-# LAB 1: Docker refresh
+# LAB 1: docker refresh
 
-In this lab we will explore the docker environment within the CDK. If you are familiar with 
+In this lab we will explore the docker environment within Minishift. If you are familiar with 
```

Comment:
i think we still want to call this the CDK. since that downstream "product" of minishift.